src/data/useChartDefinitions.js

```diff
@@ -1,20 +1,5 @@
 import { useMemo } from 'react';
+import { assignQueryAlias } from '@codeforafrica/hurumap-ui/factory/utils';
 import sectionedCharts from './charts.json';
 
-export default () =>
-  useMemo(() => {
-    // Provide the visual with unique ids for fetching
-    // The unique ids will be used to set alias in graphql
-    let index = 0;
-    sectionedCharts.forEach(x =>
-      x.charts.forEach(chart => {
-        // eslint-disable-next-line no-param-reassign
-        chart.queryAlias = `chart${index}`;
-        const { visual, stat } = chart;
-        visual.queryAlias = `viz${index}`;
-        stat.queryAlias = `viz${index}`;
-        index += 1;
-      })
-    );
-    return sectionedCharts;
-  }, []);
+export default () => useMemo(() => assignQueryAlias(sectionedCharts), []);
```

Comment:
Thinking out loud: Couldn't we have made `useMemo` part of `assignQueryAlias` or something? Unless this behaviour changes on a per app basis.